### PR TITLE
Add libdef NewRelic 4.x.x

### PR DESCRIPTION
**Pull request by facundo-viale-olx**

### CODE REVIEW

File: definitions/npm/newrelic_v4.x.x/flow_v0.53.x-/newrelic_v4.x.x.js [hunk 1]
```diff
@@ -0,0 +1,71 @@
+// @flow strict
+/* eslint-disable flowtype/no-weak-types */
+
+// @see https://docs.newrelic.com/docs/agents/nodejs-agent/supported-features/nodejs-agent-api
+declare module 'newrelic' {
+
+  declare interface TransactionHandle {
```
Comment (AndrewSouthpaw) [inline definitions/npm/newrelic_v4.x.x/flow_v0.53.x-/newrelic_v4.x.x.js]: Generally we've been finding interfaces to be less performant. Do you definitely need it to be an interface?

File: definitions/npm/newrelic_v4.x.x/test_newrelic_v4.x.x.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
+// @flow
```
Comment (AndrewSouthpaw) [inline definitions/npm/newrelic_v4.x.x/test_newrelic_v4.x.x.js]: Put the test under the versioned flow directory.

Also, use describe/it blocks.

File: definitions/npm/newrelic_v4.x.x/flow_v0.53.x-/newrelic_v4.x.x.js (hunk 1 above)
Comment (facundo-viale-olx) [reply definitions/npm/newrelic_v4.x.x/flow_v0.53.x-/newrelic_v4.x.x.js]: Oh! I wasn't aware of that. I'm moving this to a type.

File: definitions/npm/newrelic_v4.x.x/test_newrelic_v4.x.x.js (hunk 2 above)
Comment (facundo-viale-olx) [reply definitions/npm/newrelic_v4.x.x/test_newrelic_v4.x.x.js]: Done

**Comment by AndrewSouthpaw** [comment]
Cool, looking better. Did you want to change the `interface` over to types as well?

**Comment by facundo-viale-olx** [comment]
I switched `NewRelicAPI` to a type. I don't know how can i turn this into a type:

```
  declare interface Instrument {
    (opts: { moduleName: string, onRequire: () => void, onError?: (err: Error) => void }): void;
    (moduleName: string, onRequire: () => void, onError?: (err: Error) => void): void;
  }
```

**Comment by AndrewSouthpaw** [comment]
... can you not just change it to a type? Looks like it'd be fine. Essentially if you're never doing `implements Instrument` that's a pretty clear indication it doesn't need to be an interface.

**Comment by facundo-viale-olx** [comment]
Sorry my bad. I thought defining a constructor was only in a `ìnterface`, maybe I never saw a `type` defining a constructor. 
Now I know the main difference is the `implements`.

**Pull request merged**